src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -275,7 +277,16 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
 
             if (exitCode != ProcessConstants.ExitSuccess)
             {
-                logger.LogError("Build script exited with {exitCode}", exitCode);
+                logger.LogLongMessage(
```

Comment:
Currently following is the query I need to run to get details of failed build(I already saved this query for our team if you want to use):
```
traces 
| where message startswith "Build script exited with"
| project operation_ParentId
// Using leftouter here as build script output could be broken into multiple trace records based on its size
| join kind= leftouter (
   traces
   | where message startswith "Build script output" 
) on operation_ParentId
```

With my change here, it would be
```
traces
| where customDimensions.buildExitCode != 0
```